proxy/envoy/watcher.go

```diff
@@ -53,6 +55,16 @@ func NewWatcher(discovery model.ServiceDiscovery, ctl model.Controller, mesh *Me
 		return nil, err
 	}
 
+	err = ctl.AppendConfigHandler(model.RouteRule, func(model.Key, proto.Message, model.Event) { out.reload() })
```

Comment:
nit: combline line 58 and line 59 so that the scope of "err" is limited to the "if" conditional. This mirrors the style of lines 54-56 and lines 69-71.